recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "tokenizers" %}
-{% set version = "0.13.3" %}
+{% set version = "0.15.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 2e546dbb68b623008a5442353137fbb0123d311a6d7ba52f2667c8862a75af2e
+  sha256: c0a331d6d5a3d6e97b7f99f562cee8d56797180797bc55f12070e495e717c980
 
 build:
   number: 0
```

Comment:
I think it's safe to change the `skip` to `py<37`
See [here](https://github.com/huggingface/tokenizers/blob/v0.15.1/bindings/python/pyproject.toml#L3)